tendermint/src/time.rs

```diff
@@ -33,11 +33,7 @@ impl TryFrom<Timestamp> for Time {
             nanos: value.nanos,
         };
 
-        Ok(SystemTime::try_from(prost_value)
-            .map_err(|e| {
-                Kind::OutOfRange.context(format!("time before EPOCH by {} seconds", e.as_secs()))
-            })?
-            .into())
+        Ok(SystemTime::from(prost_value).into())
```

Comment:
The conversion from a prost `Timestamp` to `SystemTime` is now infallible because `prost_types::Timestamp` now provides a `Into<SystemTime>`.

Reply:
Nice!